docs/core/transactions/retry.md

```diff
@@ -1,321 +0,0 @@
----
```

Comment:
This page is super important. Is there a way we can keep this?

Reply:
added back under Advanced Concepts